supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -661,7 +661,8 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
         case _: RunStatus.Success if returnCode.isFailure =>
           FailedNonRetryableExecutionHandle(new Throwable(s"execution failed: could not parse return code as integer: " + returnCodeContents.get)).future
         case _: RunStatus.Success if !continueOnReturnCode.continueFor(returnCode.get) =>
-          FailedNonRetryableExecutionHandle(new Throwable(s"execution failed: disallowed command return code: " + returnCode.get), returnCode.toOption).future
+          val badReturnCodeMessage = s"Call ${call.fullyQualifiedName}: " + s"return code was ${returnCode.getOrElse("(none)")}"
```

Comment:
ToL:

The `returnCode.get` wasn't as horrible as it looks, since there's a case above which catches Failure.

OTOH, rolling it into the pattern match would be even nicer